src/test/java/seedu/address/logic/commands/task/MarkTaskCommandTest.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.commands.task;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalTasks.getTypicalTaskPanel;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for MarkTaskCommand.
+ */
+public class MarkTaskCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalTaskPanel(), new UserPrefs());
+
+    @Test
+    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
```

Comment:
Can add additional execute_ValidIndexUnfilteredList() test

Reply:
Noted, working on it.